README.md

```diff
@@ -1,68 +1,38 @@
 
 https://www.transifex.com/projects/p/signal-android/
 
 ## Contributing Code
-Instructions on how to setup your development environment and build Signal can be found in  [BUILDING.md](https://github.com/WhisperSystems/Signal-Android/blob/master/BUILDING.md).
-
-If you're new to the Signal codebase, we recommend going through our issues and picking out a simple bug to fix (check the "easy" label in our issues) in order to get yourself familiar. Also please have a look at the [CONTRIBUTING.md](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md), that might answer some of your questions.
-
-For larger changes and feature ideas, we ask that you propose it on the [unofficial Community Forum](https://whispersystems.discoursehosting.net) for a high-level discussion with the wider community before implementation.
-
-This repository is set up with [BitHub](https://whispersystems.org/blog/bithub/), so you can make money for committing to Signal. The current BitHub price for an accepted pull request is:
-
-[![Current BitHub Price](https://bithub.herokuapp.com/v1/status/payment/commit/)](https://whispersystems.org/blog/bithub/)
+Instructions on how to setup your development environment and build Signal can be found in  [BUILDING.md](https://github.com/LibreSignal/LibreSignal/blob/master/BUILDING.md). If you're new to the LibreSignal codebase, we recommend going through our issues and picking out a simple bug to fix (check the "easy" label in our issues) in order to get yourself familiar. Also please have a look at the [CONTRIBUTING.md](https://github.com/LibreSignal/LibreSignal/blob/master/CONTRIBUTING.md), that might answer some questions.
 
 ## Contributing Ideas
-Have something you want to say about Open Whisper Systems projects or want to be part of the conversation? Get involved in the [community forum](https://whispersystems.discoursehosting.net).
-
-## Contributing Funds
-[![Donate](https://cloud.githubusercontent.com/assets/3121306/11278543/d46e03d0-8eeb-11e5-9691-0da1bf643192.png)](https://www.coinbase.com/checkouts/51dac699e660a4d773216b5ad94d6a0b)
-
-You can add funds to BitHub to directly help further development efforts.
+Have something you want to say about LibreSignal or want to be part of the conversation? Feel invited to [post your new idea](https://github.com/LibreSignal/LibreSignal/issues/new), but please be patient for a reply and don't be mad should developers decide to not implement it or are busy with other things.
 
 Help
 ====
 ## Support
-For troubleshooting and questions, please visit our support center!
-
-http://support.whispersystems.org/
+For troubleshooting and questions, please see our [open GitHub Issues](https://github.com/LibreSignal/LibreSignal/issues?q=is%3Aopen+is%3Aissue)!
 
 ## Documentation
 Looking for documentation? Check out the wiki!
 
-https://github.com/WhisperSystems/Signal-Android/wiki
+https://github.com/LibreSignal/LibreSignal/wiki
```

Comment:
I don't think maintaining a seperate wiki makes sense.
In general, I think all paragraphs about contributing should contain a sentence like: "Before contributing to LibreSignal, think about adding your ideas to [Signal-Android](  
https://github.com/WhisperSystems/Signal-Android), as we pull from there."
Then, you could also say that general documentation and info about how to contribute is available in their wiki/md-files, but if, at some point, someone feels like writing documentation specific to the websocket support, it could be added to this repo's wiki and linked to.